omod/src/main/webapp/template/headerFull.jsp

```diff
@@ -104,7 +104,7 @@
 						<openmrs:message code="header.logged.in"/> <c:out value="${authenticatedUser.personName}" />
 					</span>
 					<span id="userLogout">
-						<a href='${pageContext.request.contextPath}/logout'><openmrs:message code="header.logout" /></a>
+						<a href='${pageContext.request.contextPath}/ms/logout'><openmrs:message code="header.logout" /></a>
```

Comment:
When this was living in Core `/ms/` was not necessary (see [here](https://github.com/openmrs/openmrs-core/blob/e40a531c6f542538bb210a06d8c588e37c47aabb/webapp/src/main/webapp/WEB-INF/template/headerFull.jsp#L99)), and the code was ported to the Legacy UI module as such.

@sherrif10 how did you figure out about this `/ms/` in the URL?

Reply:
@mks-d  thanks for your feedback on this, so i thought  the new changes i created in config.xml class would mount to the logoutservlet on that path /ms/logout hence creating a new URL, i guessed this because it was being done somewhere in fhirModule[  here ](https://github.com/openmrs/openmrs-module-fhir2/blob/master/omod/src/main/java/org/openmrs/module/fhir2/web/filter/ForwardingFilter.java#L40). More suggestions is highly needed thanks

Reply:
@sherrif10 but the pointer to the FHIR2 module hints at `/ws/` not `/ms/`, how did you get to `/ms/` then?

Did you try this at app runtime?

Reply:
@mks-d  Yes i run this app and everything was  working fine, the only problem was testing this at platform 2.4.0-snapshot which i tested  working fine also, I included /ms/logout to create a path to the url i created in config.xml similar to the fhirModule Any ideas

Reply:
@sherrif10 [this](https://github.com/openmrs/openmrs-core/blob/82c887eec226868e22e1ecfd3dcb65691a68e589/webapp/src/main/webapp/WEB-INF/web.xml#L263) is why actually.

Reply:
@mks-d  thanks for your nice explanation ,